apps/backend/api/src/app/game-resources/game-resources.service.ts

```diff
@@ -55,6 +55,30 @@ export class GameResourcesService {
     });
   }
 
+  async findRessourcesUserGame() {
+    return await this.gameResourcesRepository
```

Comment:
```suggestion
    return this.gameResourcesRepository
```
le await est inutile dans ces cas là